projects/material-dayjs-adapter/src/lib/adapter/dayjs-date-adapter.ts

```diff
@@ -203,13 +213,12 @@ export class DayjsDateAdapter extends DateAdapter<Dayjs> {
     return this.dayJs(date).isValid();
   }
 
-  // TODO: How to create an invalid Dayjs object?
   invalid(): Dayjs {
-    return;
+    return this.dayJs(null);
   }
 
-  private dayJs(input?: any, format?: string): Dayjs {
-    return this.shouldUseUtc ? dayjs.utc(input, format) : dayjs(input, format);
+  private dayJs(input?: any, format?: string, locale?: string): Dayjs {
+    return this.shouldUseUtc ? dayjs(input, { format: format, locale: locale, utc: this.shouldUseUtc }, locale).utc() : dayjs(input, { format: format, locale: locale }, locale);
```

Comment:
ooh this one gets a little sticky when we keep optional locales in mind....

i can imagine something like:
```suggestion
    if (this.shouldUseUtc && this.shouldUseLocale) {
      return dayjs(input, { format: format, locale: locale, utc: this.shouldUseUtc }, locale).utc()
    }
    if (this.shouldUseUtc) {
      return dayjs.utc(input, format);
    }
    if (this.shouldUseLocale) {
      return dayjs(input, { format: format, locale: locale }, locale);
    }
    return dayjs(input, format);
```

Reply:
See comment before: locale will always be set so it shouldn't be a problem.
If I'm not mistaken you can pass the locale parameter without including the localization plugin.
We can try that later.